packages/backend/src/shared/baseModel.test.ts

```diff
@@ -1,13 +1,15 @@
-import { BaseModel, Pagination } from './baseModel'
+import { BaseModel, Pagination, SortOrder } from './baseModel'
 
 interface PageTestsOptions<Type> {
   createModel: () => Promise<Type>
-  getPage: (pagination?: Pagination) => Promise<Type[]>
+  getPage: (pagination?: Pagination, sortOrder?: SortOrder) => Promise<Type[]>
+  sortOrder: SortOrder
 }
 
 export const getPageTests = <Type extends BaseModel>({
   createModel,
-  getPage
+  getPage,
+  sortOrder
```

Comment:
maybe there is a nice way to explicitly test sortOrder for all of the 4 cases in here (forwards/backwards x asc/desc), so that we wouldn't even need the `Math.random() < 0.5 ? SortOrder.Asc : SortOrder.Desc` line

Reply:
I am happy to do implement this. The reason that I did it randomly was just to try and keep test time in mind. But it's probably better to be thorough.

Reply:
Note that we're moving this into a separate issue.